### PR TITLE
Update Register.js

### PR DESCRIPTION
**Pull request by PanConQuesotv**
In the registry, a complete form was modified to have the title and image of the project as a whole, in which first name, last name only allows letters to be entered, and email requests that it have the structure of an at sign. 
It has a function to confirm password in which it compares so that the two are identical and it is requested that it has some requirements among these are the password must have at least eight characters it must include at least one number it must also include at least one capital letter and of a special nature, we also implement a check box for the data processing policy in which a popup is programmed to put all the information of the treatment policy aligned with ISO 27001. We also leave links for the user to consult about the documentation of This ISO and these data processing policies are obviously a prerequisite to register.

In other words, it meets the length and complexity requirements and gives clear feedback to the user, also managing to record which role is needed to enter with each of the restrictions of each role.

With permissions set correctly limiting access by role

### CODE REVIEW

**Comment by PanConQuesotv** [comment]
In the registry, a complete form was modified to have the title and image of the project as a whole, in which first name, last name only allows letters to be entered, and email requests that it have the structure of an at sign. 
It has a function to confirm password in which it compares so that the two are identical and it is requested that it has some requirements among these are the password must have at least eight characters it must include at least one number it must also include at least one capital letter and of a special nature, we also implement a check box for the data processing policy in which a popup is programmed to put all the information of the treatment policy aligned with ISO 27001. We also leave links for the user to consult about the documentation of This ISO and these data processing policies are obviously a prerequisite to register.

In other words, it meets the length and complexity requirements and gives clear feedback to the user, also managing to record which role is needed to enter with each of the restrictions of each role.

With permissions set correctly limiting access by role

**Comment by PanConQuesotv** [comment]
A registration form was implemented that includes a title and image of the project.
The fields for first and last name were modified, allowing only letters.
The email field now validates the correct structure (must contain an '@').
Added a password confirmation feature that verifies that both passwords are identical.
The password must meet the following requirements:
Minimum of eight characters.
It must include at least one number.
It must contain at least one capital letter.
It must include at least one special character.
A check box was implemented to accept the data processing policy, with a pop-up window that provides detailed information aligned to the ISO 27001 standard.
Links were added for users to consult documentation related to ISO 27001 and data processing policies, this being a prerequisite for registration."
